smart_campus/app/templates/app/manager_list_page.html

```diff
@@ -0,0 +1,86 @@
+{% extends "app/base.html" %}
+{% load static %}
+
+{% block content %}
+  <div class="ui borderless center aligned container">
+    <table class="ui selectable celled striped table">
+        <thead>
+          <tr class="center aligned">
+            <th>email</th>
+            <th>暱稱</th>
+            <th>角色</th>
+            <th>管理群組</th>
+            <th>選項</th>
+          </tr>
+        </thead>
+        <tbody>
+          {% if managers %}
+            {% for manager in managers %}
+              {% if manager.email != email %}
+                <tr class="center aligned">
+                  <td class="collapsing">{{ manager.email }}</td>
+                  <td class="collapsing">{{ manager.nickname }}</td>
+                  <td class="collapsing">{{ manager.role.name }}</td>
+                  <td class="collapsing">{{ manager.group.name }}</td>
+                  <td class="collapsing">
+                    <div class="ui mini buttons">
+                      <a class="ui basic green floated button" href="{% url 'Manager Edit Page' manager.email %}">修改</a>
+                      <a class="ui basic red floated button" href="{% url 'Manager Delete Page' manager.email %}">刪除</a>
+                    </div>
+                  </td>
+                </tr>
+              {% else %}
+                <tr class="center aligned">
+                  <td class="collapsing">{{ manager.email }}</td>
+                  <td class="collapsing">{{ manager.nickname }}</td>
+                  <td class="collapsing">{{ manager.role.name }}</td>
+                  <td class="collapsing">{{ manager.group.name }}</td>
+                  <td class="collapsing">
+                    <div class="ui mini buttons">
+                      <i class="ui basic grey floated button" style="cursor: not-allowed;">修改</i>
+                      <i class="ui basic grey floated button" style="cursor: not-allowed;">刪除</i>
+                    </div>
+                  </td>
+                </tr>
+              {% endif %}
```

Comment:
line 19~45
The "if else" block should contain only line 27,28 and line 40,41.

Reply:
Okay, fixed it.